### PR TITLE
remove python2 code

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: IPython/extensions/autoreload.py [hunk 1]
```diff
@@ -116,15 +116,14 @@
 import types
 import weakref
 from importlib import import_module
-
+from IPython.utils.py3compat import PY3
```
Comment (Carreau) [inline IPython/extensions/autoreload.py]: is `PY3` still used in this file ?

**Comment by Carreau** [comment]
Minor comment. 

+1 Thanks !

File: IPython/extensions/autoreload.py (hunk 1 above)
Comment (srinivasreddy) [inline IPython/extensions/autoreload.py]: Yes. But that will go away in the next PR.

Comment (srinivasreddy) [inline IPython/extensions/autoreload.py]: I mean the other existing PR we have for PY3

Comment (srinivasreddy) [inline IPython/extensions/autoreload.py]: I cannot find it. If you are okay, please merge. Will cleanup once all are merged .

Comment (Carreau) [inline IPython/extensions/autoreload.py]: Ah, ok. I was just waiting for test to finish to be sure my rebase was correct.

Comment (Carreau) [inline IPython/extensions/autoreload.py]: Sorry, was thinking of other PR. Merging.

**Pull request merged**